src/main/kotlin/mara/server/domain/user/User.kt

```diff
@@ -1,35 +1,38 @@
 package mara.server.domain.user
 
-import jakarta.persistence.CascadeType
+import com.fasterxml.jackson.annotation.JsonValue
 import jakarta.persistence.Entity
 import jakarta.persistence.EnumType
 import jakarta.persistence.Enumerated
 import jakarta.persistence.GeneratedValue
 import jakarta.persistence.GenerationType
 import jakarta.persistence.Id
-import jakarta.persistence.OneToMany
 import jakarta.persistence.Table
-import mara.server.domain.board.Board
 
 enum class Role { USER }
 @Entity
 @Table(name = "app_user")
 class User(
-    val name: String,
+    val nickName: String,
     val password: String,
     val kakaoId: Long?,
+    val kakaoEmail: String?,
     val googleEmail: String?,
-    val inviteCode: String
+    val inviteCode: String,
+    var profileImage: Profile,
 ) {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     val userId: Long = 0L
 
-    @OneToMany(mappedBy = "user", cascade = [CascadeType.ALL], orphanRemoval = true)
-    protected val boardMutableList: MutableList<Board> = mutableListOf()
-    val boardList: List<Board> get() = boardMutableList.toList()
-
     @Enumerated(EnumType.STRING)
     var role: Role = Role.USER
         protected set
 }
+
+enum class Profile(@JsonValue val profileValue: String) {
```

Comment:
`enum class Profile` -> `enum class ProfileImage` 가 좋을 것 같습니다! 🫡

그리고.. 코드성 데이터이니만큼 굳이 `@JsonValue val profileValue: String` 를 사용해서 소문자로 처리 하지 말고, @Enumerated(EnumType.STRING) 를 대문자 그대로 Enum을 String으로 사용하시는건 어떠신가요?

```
@Enumerated(EnumType.STRING)
var profileImage: Profile,
```

```suggestion
enum class Profile() {
    BLUE
    RED,
    YELLOW,
    GREEN,
```

Reply:
좋은것 같습니다. 변경했습니다.